### ISSUE TITLE
GitHub Action Error

### ISSUE THREAD

**Issue by jwokaty** [waldronlab/bugsigdbr#47]
The GitHub Action Workflow fails with

```
══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-ontology.R:71:9'): subsetByOntology ──────────────────────────
all(pos %in% sdf[, col]) is not TRUE

`actual`:   FALSE
`expected`: TRUE 
Backtrace:
    ▆
 1. └─bugsigdbr (local) checkSubset(sdf, "Condition", cpos[[ct]], cneg[[ct]]) at test-ontology.R:71:8
 2.   └─testthat::expect_true(all(pos %in% sdf[, col])) at test-ontology.R:27:4

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 224 ]
Error: Test failures
Execution halted
```

See https://github.com/waldronlab/bugsigdbr/actions/runs/5870373865/job/15917266645#step:22:150.

**Comment by lgeistlinger** [comment]
Thanks. I look into that. I am going to merge your pull request anyhow, as the issue here seems unrelated.

**Comment by lgeistlinger** [comment]
fixed in cbd2006b0a7912f7c8b6e2a78e919c0d0ac50324
